bin/test-run

```diff
@@ -105,16 +136,19 @@ printf "Testing Linkerd version [%s] namespace [%s] k8s-context [%s]\\n" "$linke
 # 3. if failed, exit script to avoid leaving behind stale resources which will
 # fail subsequent tests. `cleanup` is not called if this test failed so that
 # there is a chance to debug the problem
-run_upgrade_test "$linkerd_namespace"-upgrade || exit_code=$?
-if [ $exit_code -ne 0 ]; then
-  printf "\\n=== FAIL: Can't upgrade to version $linkerd_version\\n"
-  exit $exit_code
-fi
+#run_upgrade_test "$linkerd_namespace"-upgrade
+#exit_on_err "can't upgrade to version $linkerd_version"
 cleanup
 
-# run the rest of the test suite. `cleanup` is not called if this test failed
-# so that there is a chance to debug the problem
-run_test "$test_directory/install_test.go" -failfast --linkerd-namespace=$linkerd_namespace || exit_code=$?
+run_helm_test
+exit_on_err "error testing Helm"
+cleanup
```

Comment:
Probably not needed.

Reply:
As explained over slack, this is needed because `cleanup` blocks until all the resources are gone, unlike `helm delete`.